src/transformers/trainer.py

```diff
@@ -2057,6 +2066,25 @@ def _load_from_checkpoint(self, resume_from_checkpoint, model=None):
                 # release memory
                 del state_dict
                 self._issue_warnings_after_load(load_result)
+
+        # Load adapters following PR # 24096
+        elif is_peft_available() and isinstance(model, PeftModel):
+            # If train a model using PEFT & LoRA, assume that adapter have been saved properly.
+            if hasattr(model, "active_adapter") and hasattr(model, "load_adapter"):
+                if os.path.exists(resume_from_checkpoint) or os.path.exists(resume_from_checkpoint):
+                    model.load_adapter(resume_from_checkpoint, model.active_adapter)
+                    # Load_adapter has no return value present, modify it when appropriate.
+                    from torch.nn.modules.module import _IncompatibleKeys
+
+                    load_result = _IncompatibleKeys([], [])
```

Comment:
Maybe just set it to `None` and adapt `_issue_warnings_after_load` to return early if the `load_result` is `None` cause this is a bit long for nothing.

Reply:
Or maybe completely remove that line since `load_result` is not used at all after: https://github.com/llohann-speranca/transformers/blob/dae3c92d48ad8941bb502baefc06d6284498714b/src/transformers/trainer.py#L2079